From david-b@pacbell.net Thu Jun  7 14:13:36 2007
From: David Brownell <david-b@pacbell.net>
Date: Thu, 7 Jun 2007 14:13:26 -0700
Subject: USB: usb host side can be configured given PCMCIA
To: Greg KH <greg@kroah.com>, linux-usb-devel@lists.sourceforge.net
Message-ID: <200706071413.26617.david-b@pacbell.net>
Content-Disposition: inline


Platforms with PCMCIA support can implement host-side USB with "sl811_cs",
so make sure this menu shows up on platforms with PCMCIA.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/Kconfig
+++ b/drivers/usb/Kconfig
@@ -16,6 +16,7 @@ config USB_ARCH_HAS_HCD
 	boolean
 	default y if USB_ARCH_HAS_OHCI
 	default y if USB_ARCH_HAS_EHCI
+	default y if PCMCIA				# sl811_cs
 	default y if ARM				# SL-811
 	default PCI
 
